pyserver/main.py

```diff
@@ -43,24 +59,96 @@ def read_root():
 # Step 1: Comments to Topic Tree  #
 #---------------------------------#
 @app.post("/topic_tree/")
-def comments_to_tree(comments: CommentList, log_to_wandb:bool = False):
+def comments_to_tree(req: CommentsLLMConfig, log_to_wandb:bool = False):
```

Comment:
maybe this is a python idiom I'm not familiar with, but why is this param called "req"? because it's data from the API request sent from the ts server?

I don't need it changed if it's clear to others, just want clarity